docs/zh/developer/contributing.md

```diff
@@ -1,3 +1,22 @@
-# Contributing
+# 代码贡献
 Please refer to [Contribution Guideline](https://github.com/4paradigm/OpenMLDB/blob/main/CONTRIBUTING.md)
 
+## Pull Request须知

```

Comment:
=> Pull Request (PR)
then change all "pr" in this doc to "PR"; 
or you may just use the full name pull request, which is better

Reply:
later we may add a section to our CONTRIBUTING.md, then also mention this guideline in the PR template

Reply:
fix pr typo done